docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,112 @@
+
+### A new domain name
+
+A new name wouldn't be so official without a new domain,
+so we have moved home from https://material-ui.com/ to https://mui.com/, a very easy to type (and find) domain.
+
+### New logos
+
+We've tightened up the company logo to match the new brand.
+
+It's basically the same, to keep it familiar –
+we're keeping the geometrical shape, to resonate with the _building blocks_ idea of the components –
+however, we're reducing the emphasis on the 3D perspective, stepping away from the notion of elevation that Material Design coined.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/old-new.png" alt="Side-by-side of the new and old logo" style="width: 612px; margin-bottom: 16px;" />
+
+To better identify, market, and promote each product, we're introducing logos for each one of them.
+
+<img loading="lazy" src="/static/blog/material-ui-is-now-mui/products.png" alt="Logos for MUI Core, MUI X and Templates" style="width: 600px; margin-bottom: 16px;" />
+
+- **MUI Core** contains ready-to-use, forever free, foundational components.
+- **MUI X** includes Powerful and robust components for your complex apps.
+- **Templates** is a collection of fully built, out-of-the-box, React templates for your application. They are developed by the community curated by us for quality.
+
+### New package names
+
+The package names have changed in v5, which is a **breaking change**.
+You can find more details in the [release](https://github.com/mui-org/material-ui/releases/tag/v5.0.0) [notes](https://github.com/mui-org/material-ui-x/releases/tag/v4.0.0).
+The npm organization name (scope) has moved from @material-ui to [@mui](https://www.npmjs.com/org/mui).
+
+### A new website & documentation
+
+Lastly, to celebrate the start of a new chapter for the company with the release of v5.0.0, and new products in the making, we're introducing a brand new website and documentation design.
+
+You'll find a completely different theme from Material Design – new typeface, colors, box-shadows, and more,
+all done using the flexible theming features of v5.
+
+<a href="/"><img loading="lazy" src="/static/blog/material-ui-is-now-mui/palette.png" alt="Selection of the new color palette and gradients" style="width: 600px; margin-bottom: 16px;" /></a>
+
+<p class="blog-description">The color palette of our brand.</p>
+
+[Head to the new website](/).
+
+> Note that these changes do **not** impact the design of the components released under the `@mui/*` npm packages.
+
+## The path ahead
+
+There are many more exciting things to come in the future as we roll into this new chapter.
+We're grateful to have you with us, supporting the library, and helping it grow and mature.
+
+Material Design will continue to be supported with the same attention to detail as before.
+We hope to keep serving those who like the library for closely following MD, while also leveraging the strong foundation built so far to offer new components, designs, and products.
```

Comment:
The final two sentences start with we hope which reads a bit repetitive.  Ending with `We hope` seems right to me.

Maybe for this sentence:

`We strive to keep serving those who like the library for closely following MD, while also leveraging the strong foundation built so far to offer new components, designs, and products.`

or 

`We aim to keep serving those who like the library for closely following MD, while also leveraging the strong foundation built so far to offer new components, designs, and products.`

Reply:
Thanks, man. Changed it as suggested!